internal/proto/server_adaptor.go

```diff
@@ -1,6 +1,7 @@
 package proto
 
 import (
+	"github.com/windmilleng/tilt/internal/build"
```

Comment:
i'm really concerned about the dependency of the protobuf package on the build package. can we restructure the packages to avoid this dependency?

Reply:
yes